core/plugins.go

```diff
@@ -20,23 +21,50 @@ var openedFinalizePlugins map[string]Plugin
 func LoadPlugin(name string, plugintype api.PluginType, recipe *api.Recipe) (uintptr, error) {
 	fmt.Println("Loading new plugin")
 
-	localPluginPath := fmt.Sprintf("%s/%s.so", recipe.PluginPath, name)
+	projectPluginPath := fmt.Sprintf("%s/%s.so", recipe.PluginPath, name)
 
-	globalPluginPath := fmt.Sprintf("%INSTALLPREFIX%/share/vib/plugins/%s.so", name)
+	globalPluginPathsEnv, isGPPEDefined := os.LookupEnv("XDG_DATA_DIRS")
+	if !isGPPEDefined || globalPluginPathsEnv == "" {
+		globalPluginPathsEnv = "/usr/local/share:/usr/share"
```

Comment:
INSTALLPREFIX should still be included in the search path, no matter if XDG_DATA_DIRS is defined or not, the order should be `<project>/plugins, INSTALLPREFIX/share/vib/plugins, <xdg data dirs>/vib/plugins`